### DISCUSSION TITLE
'strapi' is not recognized as an internal or external command, operable program or batch file. showing this error while using strapi develop command.

### DISCUSSION THREAD

**Discussion by shrutisahatpure** [strapi/strapi#6898] [Legacy Topics]
![Untitleds](https://user-images.githubusercontent.com/50610929/86436875-d3e7ac00-bd20-11ea-9ef8-1c97ba4234b3.png)

**Comment by MattieBelt** [comment]
Duplicate [#6899 ](https://github.com/strapi/strapi/discussions/6899#discussioncomment-33402)
